terms.html

```diff
@@ -114,9 +122,12 @@
 The ability of a <a>holder</a> to make fine-grained decisions about what
 information to share.
   </dd>
-  <dt><dfn data-lt="subjects|subject's">subject</dfn></dt>
-  <dd>
-A thing about which <a>claims</a> are made.
+  <dt><dfn data-lt="subjects|subject's|subject(s)">subject</dfn></dt>
+The <a>entity</a> about which a <a>claim</a> is made.
```

Comment:
Per @jandrieu 's point here:

https://github.com/w3c/vc-data-model/pull/1172#discussion_r1281250448

Claims are about subjects, or made by subjects... we seem to be making this language worse over time.

Reply:
I don't see any prior instance of "Claims are ... made by subjects", and I cannot think of any instance where it would be correct. Claims are made *about* subjects, full stop.

Reply:
I mean in the sense of RDF Statements...

```
<subject> <predicate> <object>
```

```
credential-id (subject) issuer (predicate / claim) issuer-id (subject)
```

Reply:
> I mean in the sense of RDF Statements...
> 
> ```
> <subject> <predicate> <object>
> ```
> 
> ```
> credential-id (subject) issuer (predicate / ***claim***) issuer-id (subject [sic])
> ```

(Emphasis mine) This is not the VCDM's definition of a claim. The claim is the full sentence or statement; the full expression of `<subject> <predicate> <object>`, it is not just the predicate or property:

```
claim: credential-id (subject) issuer (predicate / property) issuer-id (value / object)
```

Reply:
^ this is exactly the kind of thing that belongs in the "value of JSON-LD section", feel free to comment here: https://github.com/w3c/vc-data-model/issues/1227

The reason it belongs there... is that nobody will see a "claim" by your definition, until after the normative `@context` is applied... which means that a JSON-LD Verifiable Credential never contains "claims" by your definition...

![Screen Shot 2023-08-04 at 5 35 59 PM](https://github.com/w3c/vc-data-model/assets/8295856/4c7197b2-d806-4a6e-ae2a-c68db4ef863d)

The visual examples define claim without expressing the predicate as an IRI... but you are asserting that the predicate must be an IRI, in order for a complete statement to be called a "claim"...

Also you can see from the visual examples that "subjects make claims about objects".. https://w3c.github.io/vc-data-model/#claims

Reply:
> which means that a JSON-LD Verifiable Credential never contains "claims" by your definition...

Yes it does. It's just expressed in a syntax different from N-Quads, whatever pseudo syntax we were using above, or the pictures linked above. It's expressed in the syntax: JSON-LD. This is how you express a statement in JSON-LD:

```js
{
  "id": "the subject",
  "the_property": "the_value"
}
```

That it doesn't look like some other syntax or pictorial representation does not change that fact. All of these things express the same thing using different formats. The JSON-LD spec *defines* how to express such statements *in JSON*. That's what that spec does. This spec (VCDM) leverages that work so we don't have to repeat ourselves -- and says how to express information about "who" is making the claims (the "issuer") and so on.

Reply:
> The visual examples define claim without expressing the predicate as an IRI... but you are asserting that the predicate must be an IRI, in order for a complete statement to be called a "claim"...

That's not what I was asserting above -- I was asserting that a "property" is not a "claim". The claim is the entire triple: "subject - property - value". It is not merely a component of that statement, it is the entire statement.

Ensuring there's a way to be globally unambiguous with property names is an orthogonal issue.

Reply:
> Also you can see from the visual examples that "subjects make claims about objects".. https://w3c.github.io/vc-data-model/#claims

No, I do not see that. I see that claims are made about subjects. When the picture says "Pat knows Sam", that is not a claim made by Pat. Is that how you're reading it? Is that the source of confusion perhaps?

Reply:
I've made some more changes to make things compatible with RDF (as per its Primer)